.eslintrc.json

```diff
@@ -0,0 +1,39 @@
+{
+    "env": {
+      "es2021": true,
+      "node": true
+    },
+    "extends": [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "ecmaVersion": 12,
+      "sourceType": "module"
+    },
+    "plugins": [
+      "@typescript-eslint"
+    ],
+    "rules": {
+      "@typescript-eslint/no-unused-vars": [
+        "warn",
+        {
+          "argsIgnorePattern": "^_"
+        }
+      ],
+      "@typescript-eslint/no-namespace": "off",
+      "@typescript-eslint/no-inferrable-types" : "off",
+      "@typescript-eslint/no-explicit-any" : "off",
```

Comment:
Hmm. Okay. Maybe not worth doing in the initial PR, but it would be nice to have at some point. Would you be able to create an issue for it once this PR is merged?